src/main/kotlin/game/racingcar/view/StandardInputView.kt

```diff
@@ -1,13 +1,28 @@
 package game.racingcar.view
 
+import game.racingcar.util.requireOrThrow
+
 class StandardInputView : InputView {
-    override var numberOfCar: Int = 0
-    override var numberOfLap: Int = 0
+    private lateinit var carNames: List<String>
+    private var numberOfLap: Int = 0
```

Comment:
```suggestion
    val carNames : List<String> by lazy { carNames() }
    val numberOfLap: Int by lazy { numberOfLap() }

    override fun numberOfLap(): Int {
        println("시도할 횟수는 몇 회인가요?")
        return readln().toInt()
    }

    override fun carNames(): List<String> {
        println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).")
        return readln().split(",")
            .requireOrThrow("자동차 이름은 빈 값일 수 없습니다.") { it.isNotBlank() }
            .map { it.take(NAME_MAX_LENGTH) }
    }
```

 초기화 과정에 자원을 많이 사용하거나, 객체를 사용할 때마다 초기화하지 않아도 되는 객체에 대해서는 지연 초기화(lazy initialization) 패턴을 사용할 수 있을 것 같아요.

참고
- [lazy-properties](https://kotlinlang.org/docs/delegated-properties.html#lazy-properties)